src/main/java/com/blackducksoftware/integration/hub/alert/web/actions/distribution/DistributionConfigActions.java

```diff
@@ -129,6 +140,64 @@ public String validateConfig(final R restModel) throws AlertFieldException {
         return Collections.emptyList();
     }
 
+    protected void saveConfiguredProjects(final CommonDistributionConfigEntity commonEntity, final R restModel) {
+        if (Boolean.TRUE.equals(commonEntity.getFilterByProject())) {
+            final List<String> configuredProjectsFromUI = restModel.getConfiguredProjects();
+            if (configuredProjectsFromUI != null) {
+                // Remove the old DistributionProjectRelations for the distribution configuration:
```

Comment:
This comment and the one below it are describing something that can be explained through a method name. I'd recommend adding a remove and add project helper methods.